src/Kernel-Tests-Extended/ProcessTest.class.st

```diff
@@ -7,6 +7,36 @@ Class {
 	#category : #'Kernel-Tests-Extended-Processes'
 }
 
+{ #category : #running }
+ProcessTest >> setUp [ 
+	super setUp.
+	
+	self executionProcessMonitor disable
```

Comment:
Process monitoring of TestExecutionEnvironment should be disabled because tests here are about methods involving into monitoring logic.